### PR TITLE
Finish mobile version: Works and About sections

### PR DESCRIPTION
**Pull request by DamianDib9**
### **_Project requirements_**
________________

 Build the middle section of the mobile website. We included a screenshot of Template 1 below for reference, but you should stick to the template that you chose. This sections of the mobile website should include:
  - "Works" section✅
  - The "About myself" section (placed after the projects and before the contact form): About myself✅

- Make sure the content of the "About myself" section is aligned with flexbox.✅
- Use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.✅
- Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.✅
- Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.✅
- Implement the button interactions (enable, hover, etc.).✅
 The website should be responsive, following these breakpoints:
  - Minimum mobile screen size should be 375px✅
  - Minimum desktop screen size should be 768px✅

Hope you like it! 😃👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,14 +31,164 @@ <h1 class="mainTitle">I’m Damian.
...
+
+                    <li><a href="https://www.google.com/"><img class="iconList" src="./img/icon/mlogo.svg"
+                                alt="mlogo"></a>
+                    </li>
+
+
+                </ul>
+                <button class="seeProjectButton">Get my resume</button>
+
+                <div class="languages">
```
Comment (topeogunleye) [inline index.html]: Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

## Solution with a detailed demonstration:-
Following is the code representation of the above if you are confused about what the instructions mean:-


```
                <ul class="languages">
                    <div class="techTitles">
                        <h4>Languages</h5> <img src="./img/tech logos/arrow down.svg" alt="arrow">
                    </div>
                    <li>
                    <ul class="desplTechList">
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/js.svg" alt="JavasCript">
                            <h5 class="techText">JavaScript</h5>
                        </li>
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/html.svg" alt="HTML">
                            <h5 class="techText">HTML</h5>
                        </li>
                        <li class="techDescr"><img class="logoPadding" src="./img/tech logos/css.svg" alt="CSS">
                            <h5 class="techText">CSS</h5>
                        </li>
                      </ul>
                    </li>
                    <li>
                        <ul class="techTitles" class="underline">
                            <h4>Frameworks</h5> <img src="./img/tech logos/arrow right.svg" alt="arrow">
                        </ul>
                    </li>
                    <li>
                        <ul class="techTitles" class="underline">
                            <h4 >Skills</h5> <img src="./img/tech logos/arrow right.svg" alt="arrpw">
                        </ul>
                    </li>
                </ul>
```

File: style.css [hunk 2]
```diff
@@ -61,3 +73,176 @@ h3 {
...
+  color: #6070ff;
+  padding-top: 0.3rem;
+  padding-bottom: 0.3rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 8px;
+  font-weight: 500;
+}
+
+.seeProjectButton {
```
Comment (topeogunleye) [inline style.css:163]: - [x] Great job adding the hover and enabled buttons interactions, however, please kindly add the active (when it's pressed) interaction as shown in the Figma template below:

Screenshot from 2022-04-27 19-12-50

the button on the pressed state has the `background: #2230D2;`

### EXAMPLE:
**for pressed state**
```
.seeProjectButton:active {
   background-color: #2230D2;
}
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @DamianDib9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
Great job! Here's a summary of your project review:

**Works section**
- The container of the section is a section tag 🆗
- The main container of the projects has a display:grid 🆗
- The section has the proper background image (decorations are always backgrounded images, not img tags) 🆗
- The title (e.g., "My Recent Works") uses a header tag hx 🆗

For each project:
- The content of the project is aligned with the flexbox 🆗
- The project title (e.g., "Tonic") use a header tag hx 🆗
- The project has an image, that could use an img or use create it as a background 🆗
- The paragraph (e.g., "A daily section ...") is a p tag 🆗
- The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design) 🆗
- The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list ul and each of them a li 🆗

- The section has the margins (top, bottom, left, right) defined in the design 🆗
- The section uses the font and colours (background and font) defined for the template 🆗

**About myself section**
- The container of the section is a section tag 🆗
- The section has the proper background image (decorations are always backgrounded images, not img tags) 🆗
- The title (e.g., "About myself") uses a header tag hx 🆗

If there are other titles (e.g., "Let's connect") use a header tag hx:
- The paragraph (e.g., "I'm a software ...") is a p tag 🆗

The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag):
- The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Hover and Pressed 🆗

If the section has a navigation link (e.g., "Work") it uses an a tag:
- The content is aligned with flexbox 🆗
- The section has the margins (top, bottom, left, right) defined in the design 🆗
- The section uses the font and colours (background and font) defined for the template 🆗

👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (DamianDib9) [reply style.css:163]: OK! Done.
image

File: index.html (hunk 1 above)
Comment (DamianDib9) [reply index.html]: OK! Thanks for the detailed description.  Done!

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @DamianDib9 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Well structured files :white_check_mark:
  * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] N/A Keep doing the great work

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**